### ISSUE TITLE
Configure cannot detect build/host/target on Windows

### ISSUE THREAD

**Issue by lsorber** [coin-or/Cbc#6]
Using the MSYS2 toolchain, `configure` (specifically, `config.guess`) cannot detect the build/host/target on Windows. A workaround is to manually specify `--build=x86_64-w64-mingw32`.

Related to #5 in the sense that using a more recent version of Autotools will probably fix the problem.

**Comment by tkralphs** [comment]
Yes, this is a known issue and the workaround is the only one that I know of at this point, but we are working on upgrading the autotools version (this is unfortunately a big task).
